README.md

```diff
@@ -41,6 +41,16 @@ Some applications load from `.env`, `.env.local`, `.env.development` and `.env.d
 (see [#37](https://github.com/entropitor/dotenv-cli/issues/37) for more information).
 `dotenv-cli` supports this using the `-c` flag for just `.env` and `.env.local` and `-c development` for the ones above.
 
+### Override
+Override any environment variables that have already been set on your machine (or in case of using multiple environment files, defined by previous environment file) with values from your .env file.
+
+Example:
+```bash
+$ echo "API_TOKEN=default" > .env
+$ echo "API_TOKEN=secret" > .env.override # .env.override may be ignored by .gitignore
+$ dotenv --override -e .env -e .env.override -p API_TOKEN
+secret
```

Comment:
This is a bad example because `--override` should not be used for this as the example below can be used just as well

```bash
$ echo "API_TOKEN=default" > .env
$ echo "API_TOKEN=secret" > .env.override # .env.override may be ignored by .gitignore
$ dotenv -e .env.override -e .env -p API_TOKEN
secret
```

This would be a better example for this, I guess
```suggestion
$ echo "API_TOKEN=secret" > .env
$ API_TOKEN="system-default" dotenv --override -e .env -p API_TOKEN
secret
```